diff --git a/node_modules/react-native-screens/lib/typescript/types.d.ts b/node_modules/react-native-screens/lib/typescript/types.d.ts
index 35369eb..e2e5100 100644
--- a/node_modules/react-native-screens/lib/typescript/types.d.ts
+++ b/node_modules/react-native-screens/lib/typescript/types.d.ts
@@ -18,8 +18,8 @@ export declare type GestureResponseDistanceType = {
     bottom?: number;
 };
 export interface ScreenProps extends ViewProps {
-    active?: 0 | 1 | Animated.AnimatedInterpolation;
-    activityState?: 0 | 1 | 2 | Animated.AnimatedInterpolation;
+    active?: 0 | 1 | Animated.AnimatedInterpolation<any>;
+    activityState?: 0 | 1 | 2 | Animated.AnimatedInterpolation<any>;
     children?: React.ReactNode;
     /**
      * Boolean indicating that swipe dismissal should trigger animation provided by `stackAnimation`. Defaults to `false`.
