diff --git a/node_modules/react-native-screens/lib/typescript/types.d.ts b/node_modules/react-native-screens/lib/typescript/types.d.ts
index bcfb012..0c1be14 100644
--- a/node_modules/react-native-screens/lib/typescript/types.d.ts
+++ b/node_modules/react-native-screens/lib/typescript/types.d.ts
@@ -19,8 +19,8 @@ export type GestureResponseDistanceType = {
     bottom?: number;
 };
 export interface ScreenProps extends ViewProps {
-    active?: 0 | 1 | Animated.AnimatedInterpolation;
-    activityState?: 0 | 1 | 2 | Animated.AnimatedInterpolation;
+    active?: 0 | 1 | Animated.AnimatedInterpolation<any>;
+    activityState?: 0 | 1 | 2 | Animated.AnimatedInterpolation<any>;
     children?: React.ReactNode;
     /**
      * Boolean indicating that swipe dismissal should trigger animation provided by `stackAnimation`. Defaults to `false`.
