diff --git a/node_modules/@types/node/timers.d.ts b/node_modules/@types/node/timers.d.ts
index 039f31f..510b722 100644
--- a/node_modules/@types/node/timers.d.ts
+++ b/node_modules/@types/node/timers.d.ts
@@ -23,7 +23,6 @@ declare module "timers" {
          */
         ref?: boolean | undefined;
     }
-    let setTimeout: typeof global.setTimeout;
     let clearTimeout: typeof global.clearTimeout;
     let setInterval: typeof global.setInterval;
     let clearInterval: typeof global.clearInterval;
@@ -130,33 +129,6 @@ declare module "timers" {
                 [Symbol.dispose](): void;
             }
         }
-        /**
-         * Schedules execution of a one-time `callback` after `delay` milliseconds.
-         *
-         * The `callback` will likely not be invoked in precisely `delay` milliseconds.
-         * Node.js makes no guarantees about the exact timing of when callbacks will fire,
-         * nor of their ordering. The callback will be called as close as possible to the
-         * time specified.
-         *
-         * When `delay` is larger than `2147483647` or less than `1`, the `delay`will be set to `1`. Non-integer delays are truncated to an integer.
-         *
-         * If `callback` is not a function, a `TypeError` will be thrown.
-         *
-         * This method has a custom variant for promises that is available using `timersPromises.setTimeout()`.
-         * @since v0.0.1
-         * @param callback The function to call when the timer elapses.
-         * @param [delay=1] The number of milliseconds to wait before calling the `callback`.
-         * @param args Optional arguments to pass when the `callback` is called.
-         * @return for use with {@link clearTimeout}
-         */
-        function setTimeout<TArgs extends any[]>(
-            callback: (...args: TArgs) => void,
-            ms?: number,
-            ...args: TArgs
-        ): NodeJS.Timeout;
-        // util.promisify no rest args compability
-        // eslint-disable-next-line @typescript-eslint/no-invalid-void-type
-        function setTimeout(callback: (args: void) => void, ms?: number): NodeJS.Timeout;
         namespace setTimeout {
             const __promisify__: typeof setTimeoutPromise;
         }
